[Revised] Adding MarkBind as an alternative docs option

damithc opened the pull request:
A revised version based on #156 
Once finalized, all commits will be squashed to one commit and the commit message will mention #156 as the primary source of changes
Deployed site https://damithc.github.io/ab3-markbind/index.html

damithc commented:
@tlylt can have a look?

tlylt commented on .github/workflows/docs.yml:
```diff
@@ -0,0 +1,25 @@
+          token: ${{ secrets.GITHUB_TOKEN }}
+          rootDirectory: './docs'
+          baseUrl: '/addressbook-level3' # replace with your repo name
+          version: '^5.0.2'
```
```suggestion
          version: '^5.1.0'
```

tlylt commented:
> @tlylt can have a look?

Other than the above, LGTM!

Commit:
Migrate docs to MarkBind

Let's migrate the docs site from Jekyll to MarkBind.

Primary author: @tlylt in se-edu/addressbook-level3/pull/156
Further tweaks: @damithc in se-edu/addressbook-level3/pull/206

damithc commented:
Released to students as an option (see step 4 [here](https://nus-cs2103-ay2324s1.github.io/website/admin/appendixE-gitHub.html#:~:text=the%20developers%20team.-,tP%20Team%20Repo%20Setup,-(/)%20Only))
Closing this PR as it is never meant to be merged. We can observe how things go this semester and revise further in the following semester.

Thanks again @tlylt for your immense help with this.  💯
